src/models/searchservice/request/StaticFilter.ts

```diff
@@ -5,9 +5,49 @@ import { FieldValueFilter } from './FieldValueFilter';
  *
  * @public
  */
-export type StaticFilter =
-  { kind: 'fieldValue', value: FieldValueFilter } |
-  { kind: 'combination', combinator: FilterCombinator, children: StaticFilter[] };
+export type StaticFilter = FieldValueStaticFilter | DisjunctionStaticFilter | ConjunctionStaticFilter;
+
+/**
+ * A {@link FieldValueFilter} with the kind of filter specified
+ * to discriminate between static filter types.
+ *
+ * @public
+ */
+export interface FieldValueStaticFilter extends FieldValueFilter {
+  /** The kind of static filter. */
+  kind: 'fieldValue'
+}
+
+/**
+ * A static filter composed by combining filters with the logical OR
+ * operator. The combined filters can either be field value filters or
+ * other disjunction filters.
+ *
+ * @public
+ */
+export interface DisjunctionStaticFilter {
+  /** {@inheritDoc FieldValueStaticFilter.kind} */
+  kind: 'disjunction',
+  /** {@inheritDoc FilterCombinator.OR} */
+  combinator: FilterCombinator.OR,
+  /** The filters to combine together. */
+  filters: (DisjunctionStaticFilter | FieldValueStaticFilter)[]
```

Comment:
So an $or filter is not able to have any $and's inside of it?

Reply:
Exactly! That would be a disjunction of conjunctions, which is not allowed

Reply:
Wow I really thought they would allow for a single layer, but now I remember you said you switched a single $and and $or and it broke